Formula/freerdp.rb

```diff
@@ -1,8 +1,8 @@
 class Freerdp < Formula
   desc "X11 implementation of the Remote Desktop Protocol (RDP)"
   homepage "https://www.freerdp.com/"
-  url "https://github.com/FreeRDP/FreeRDP/archive/2.0.0-rc4.tar.gz"
-  sha256 "3406f3bfab63f81c1533029a5bf73949ff60f22f6e155c5a08005b8b8afe6d49"
+  url "https://pub.freerdp.com/releases/freerdp-2.0.0.tar.gz"
```

Comment:
Why is it moving?

Reply:
It prpobably should have been from here in the first place, but there wasn't clear directions until now.

The release page says:

> See https://pub.freerdp.com/releases/ for the release tarball.